tests/mdev_configuration_allocation_test.go

```diff
@@ -214,7 +213,6 @@ var _ = Describe("[Serial][sig-compute]MediatedDevices", Serial, decorators.VGPU
 
 			By("Creating a configuration for mediated devices")
 			config = kv.Spec.Configuration
-			config.DeveloperConfiguration.FeatureGates = append(config.DeveloperConfiguration.FeatureGates, virtconfig.GPUGate)
 			config.MediatedDevicesConfiguration = &v1.MediatedDevicesConfiguration{
```

Comment:
Out of scope for this PR but I wonder if we can make this setting a default for kubevirtci and move these tests in the parallel lane..?

@xpivarc @fossedihelm

Reply:
IDK if we can make such an assertion. Maybe @dhiller @brianmcarey can help here.
Consider that these tests only run in vgpu lane

Reply:
I'm wondering if these tests would work on any hardware. 
Should we consider adding some decorators for these tests?